tests/js/server/dump/dump-mixed-cluster.js

```diff
@@ -0,0 +1,1012 @@
+/*jshint globalstrict:false, strict:false, maxlen : 4000 */
```

Comment:
This file is again a quite large copy/paste. Please re-use functionality. Also, there is lots of unused code. And again only two tests here are related. So I'll not review all available tests here. 

There needs to be a better way to define a new suite here while at the same time being able to reuse our already existent code.